docs/streams/upgrade-guide.html

```diff
@@ -35,7 +35,7 @@ <h1>Upgrade Guide and API Changes</h1>
 
     <p>
         Upgrading from any older version to {{fullDotVersion}} is possible: you will need to do two rolling bounces, where during the first rolling bounce phase you set the config <code>upgrade.from="older version"</code>
-        (possible values are <code>"0.10.0" - "2.3"</code>) and during the second you remove it. This is required to safely upgrade to the new cooperative rebalancing protocol of the embedded consumer. Note that you will remain using the old eager
+        (possible values are <code>"0.10.0" - "2.4"</code>) and during the second you remove it. This is required to safely upgrade to the new cooperative rebalancing protocol of the embedded consumer. Note that you will remain using the old eager
```

Comment:
`2.4` is now the possible old version value

Reply:
I think 2.4 already launches the incremental rebalancing, cc @ableegoldman

Reply:
not sure about this as it describes a rolling upgrade from an older version, so we'll need @ableegoldman to confirm

Reply:
@showuon Boyang is right, there is no `upgrade.from` config for 2.4 since that's when cooperative rebalancing was enabled. So if you upgrade to 2.5 from any version lower than 2.4, you will need to go through this upgrade path and set the config.

Reply:
Thank you, @ableegoldman @abbccdda @bbejeck , I've reverted back this version to `2.3` now in this commit https://github.com/apache/kafka/pull/8622/commits/460768e71f5c7d427a6faffdded9b0478ade1db1. Thank you very much!